Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/MousePointer.cs

```diff
@@ -75,15 +82,27 @@ public override IMixedRealityController Controller
         /// <inheritdoc />
         public override void OnPreSceneQuery()
         {
+            // screenspace to ray conversion
             transform.position = CameraCache.Main.transform.position;
 
-            Ray ray = new Ray(Position, Rotation * Vector3.forward);
+            Ray ray = new Ray(transform.position, transform.forward);
             Rays[0].CopyRay(ray, PointerExtent);
 
             if (MixedRealityRaycaster.DebugEnabled)
             {
                 Debug.DrawRay(ray.origin, ray.direction * PointerExtent, Color.green);
             }
+
+            // ray to worldspace conversion
+            gameObject.transform.position = transform.position + transform.forward * DefaultPointerExtent;
+        }
+
+        public override Vector3 Position
+        {       
+            get
+            {
+                return gameObject.transform.position;
```

Comment:
>gameObject.transform.position [](start = 23, length = 29)

By overriding the base implementation here we are ignoring raycastOrigin. I think we could be better off having mouse pointer not derive from BaseControllerPointer / ControllerPoseSynchronizer.